meson_options.txt

```diff
@@ -90,3 +88,19 @@ option('openbmc-flash-path', type: 'string', value: '/run/initramfs',
        description: 'The path where from the OpenBMC image will be flashed.')
 option('openbmc-whitelist-file-name', type: 'string', value: 'whitelist',
        description: 'The OpenBMC whilte file name.')
+
+option('tar-cmd', type: 'string', value: '/bin/tar',
+       description: 'command line tool to work with TAR archieve')
+option('reboot-cmd', type: 'string', value: '/sbin/reboot',
+       description: 'command line tool to reboot system')
+option('fw_setenv-cmd', type: 'string', value: '/sbin/fw_setenv',
```

Comment:
I prefer to use hyphens instead of underscore in the option name.
Could you please do not mix the styles?

Reply:
this was a command name, but I can replace, no problem

Reply:
I'm fine with underscores in the options if they are parts of command names